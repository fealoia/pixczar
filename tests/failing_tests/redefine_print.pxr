Int x = 0;
Void print(){
	x = 1;
}

Void main() {
	print();
}
