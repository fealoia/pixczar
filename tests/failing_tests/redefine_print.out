Fatal error: exception Failure("expecting 1 argument in print")
